digid_eherkenning/locale/nl/LC_MESSAGES/django.po

```diff
@@ -383,18 +387,30 @@ msgstr "eIDAS attribute consuming service index"
 msgid "Attribute consuming service index for the eIDAS service"
 msgstr "Attribute consuming service index voor de eIDAS-service"
 
-#: models/eherkenning.py:71 models/eherkenning.py:79
+#: models/eherkenning.py:68
 msgid "eIDAS service UUID"
 msgstr "UUID eIDAS-service"
 
-#: models/eherkenning.py:74
+#: models/eherkenning.py:76
+msgid "eIDAS service instance UUID"
+msgstr "UUID eIDAS-service instance"
+
+#: models/eherkenning.py:71
 msgid ""
 "UUID of the eIDAS service. Once entered into catalogues, changing the value "
 "is a manual process."
 msgstr ""
 "UUID van de eIDAS-service. Eenmaal dit in catalogi opgenomen is kan de "
 "waarde enkel via een handmatig proces gewijzigd worden."
 
+#: models/eherkenning.py:79
+msgid ""
+"UUID of the eIDAS service instance. Once entered into catalogues, "
+"changing the value is a manual process."
+msgstr ""
+"UUID van de eIDAS-service-instantie. Eenmaal dit in catalogi opgenomen "
+"is kan de waarde enkel via een handmatig proces gewijzigd worden."
```

Comment:
Ik vind dit niet echt lekker lopen: eigenlijk hoort er een komma tussen "is" en "kan", en de taal is wat formeel. Maar het is gekopieerd van bestaande strings. Dus prima.